contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -427,8 +427,9 @@ contract FunctionsRouter is IFunctionsRouter, FunctionsSubscriptions, Pausable,
       }
       // call and report whether we succeeded
       // call(gas,addr,value,argsOffset,argsLength,retOffset,retLength)
+      let gasBeforeCall := gas()
```

Comment:
IIUC, this change corrects for the fact that previously we were charging users for the gas used by lines 412-430, which should just be included in gasOverhead. Is this correct?

Reply:
Yes exactly